apps/console/java/org.wso2.identity.apps.console.server.feature/resources/deployment.config.json.j2

```diff
@@ -1421,6 +1421,36 @@
                 }
             },
             {% endif %}
+            {% if console.sms_templates is defined %}
+            "smsTemplates": {
```

Comment:
Can u mention the related framework PR here?

Reply:
https://github.com/wso2/carbon-identity-framework/pull/6049